packages/docs/docs/concepts/cornerstone-tools/annotation/visibility.md

```diff
@@ -0,0 +1,42 @@
+---
+id: visibility
+title: Visibility
+---
+
+# Visibility
+
+Annotations can have their visibility changed. You can use
+the visibility API to show/hide annotations.
+
+## API
+
+There are various APIs for showing and hiding annotations along with get/set methods
+
+```js
+import { annotations } from '@cornerstonejs/tools';
+
+// changing an annotation visibility to be visible (implicit visible param).
+annotations.visibility.setAnnotationVisibility(annotationUID);
+
+// changing an annotation visibility to NOT be visible.
+annotations.visibility.setAnnotationVisibility(annotationUID, false);
+
+// get all the hidden annotations uids.
+annotations.visibility.getAnnotationUIDsHidden();
```

Comment:
hmm, not sure if it is me only, but It feels weird that the API has both `visible` and `hidden` methods. Can't we say `getAnnotationUIDsInvisible`? or `getAnnotationUIDsInvisibleCount`? This might be only me since I'm not well known for naming things :D @JamesAPetts

Reply:
This is not a requirement on my impl. So I will remove it. 
If there is a need for something like that in future we can add it back.